lib/index.js

```diff
@@ -1,8 +1,13 @@
 'use babel';
 
+import * as fs from 'fs';
```

Comment:
I changed out all the imports for require calls in the places where they are needed. It looks a little odd, but it shaved about 20-30ms off the load time.